galasa-ui/src/app/auth/tokens/route.ts

```diff
@@ -8,11 +8,13 @@ import { createAuthenticatedApiConfiguration } from '@/utils/api';
 import AuthCookies from '@/utils/authCookies';
 import { cookies } from 'next/headers';
 import { NextRequest, NextResponse } from 'next/server';
-import { AuthenticationAPIApi } from '@/generated/galasaapi';
+import { AuthenticationAPIApi, UsersAPIApi } from '@/generated/galasaapi';
 
 // Stop this route from being pre-rendered
 export const dynamic = 'force-dynamic';
 
+const CLIENT_API_VERSION = "0.37.0"
```

Comment:
Perhaps we should put this somewhere common and import it here since other REST API calls may need to supply the webui's API version.

Also, this should now be 0.38.0